mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
```

Comment:
Hardcoded error message "Order not found" should be replaced with a more descriptive error message that provides additional context about the specific error that occurred. This will help developers quickly identify and address the root cause of the problem. Consider using error codes or logging detailed error information to enhance maintainability and debugging process.